### PR TITLE
Add segments to firefox-suggest-history-vs-offline

### PR DESCRIPTION
**Pull request by rebecca-burwei**
Might have invalid parameters for window_start and window_end, but I want to give it a try.

### CODE REVIEW

**Comment by rebecca-burwei** [comment]
Hi @scholtzan , when can I expect to see these changes on https://protosaur.dev/partybal/firefox_suggest_history_vs_offline.html ?

**Comment by scholtzan** [comment]
Within a few hours. Jetstream needs to rerun the analysis (which takes about 15 minutes) and then the vizualization needs to get updated (which happens every 3 hours). I can also trigger the vizualization update manually once the analysis results are there if you need the results asap.

**Comment by rebecca-burwei** [comment]
@scholtzan no hurry, I can wait. Thanks for the info :)

**Comment by rebecca-burwei** [comment]
Hi @scholtzan , in this PR, I'm adding segments to the custom config, but I haven't seen those segments appear on the [front end]( https://protosaur.dev/partybal/firefox_suggest_history_vs_offline.html ) yet. Do you know what I might have done wrong? Are cross joins allowed in the from expression? Do I need to register the segment with each metric?

File: firefox-suggest-history-vs-offline.toml [hunk 1]
```diff
@@ -1,5 +1,43 @@
-[metrics]
+[experiment]
+Segments = ['urlbar_exposed', 'urlbar_users']
```
Comment (scholtzan) [inline firefox-suggest-history-vs-offline.toml:2]: Looks like there is a typo here
```suggestion
segments = ['urlbar_exposed', 'urlbar_users']
```

**Comment by scholtzan** [comment]
There is a typo in `Segments` which should be `segments`

**Comment by scholtzan** [comment]
It's interesting that this didn't throw an error when validating the config file. I'll see if I can add some additional checks here to so that errors get thrown for typos in the future.

**Comment by rebecca-burwei** [comment]
@scholtzan thank you! I'll update

**Pull request merged**